host-configs/docker/clang@10.0.0.cmake

```diff
@@ -4,10 +4,14 @@
 # CMake executable path: /usr/bin/cmake
 #------------------------------------------------------------------------------
 
+set(CMAKE_PREFIX_PATH "/home/axom/axom_tpls/clang-10.0.0/umpire-2022.10.0-poiqu74osyxwoxcwzlauyzu6q6pom7dd;/home/axom/axom_tpls/clang-10.0.0/raja-2022.10.5-nk2szvoktgmxnynrjlo4ihrof5fskg2b;/home/axom/axom_tpls/clang-10.0.0/camp-2022.10.1-kt6bbtmyqcycglsdrmmdmjjrzhn4cbmu;/home/axom/axom_tpls/clang-10.0.0/mfem-4.5.2-vlkan3esoqo4ext3qsa6m3qiau3gcetk;/home/axom/axom_tpls/clang-10.0.0/hypre-2.24.0-a7pavarlyr7ao3qlmwxbayorpwhmoazm;/home/axom/axom_tpls/clang-10.0.0/lua-5.4.4-5w3c5kbxtpxmoxcljzuxoczixmr7vlve;/home/axom/axom_tpls/clang-10.0.0/readline-8.2-gxedy6cgf5sabngl37xihiqosh4qqodd;/home/axom/axom_tpls/clang-10.0.0/ncurses-6.4-kpi3vqzy2j4fxy3bdhb5avsetkvvf65c;/home/axom/axom_tpls/clang-10.0.0/conduit-0.8.8-frwcazx76ivrq337w4i3sjgyuvavl7f3;/home/axom/axom_tpls/clang-10.0.0/parmetis-4.0.3-qb5ova7xrfnnjcnwjg277camap6xfu7d;/home/axom/axom_tpls/clang-10.0.0/metis-5.1.0-xlsz5xtc5npvrzlfkwjiho6uuotdmhv6;/home/axom/axom_tpls/clang-10.0.0/hdf5-1.8.22-u6laljs32gtrqbuc2tysixkb5vj6qypw;/home/axom/axom_tpls/clang-10.0.0/zlib-1.2.13-rgebxh4wloej77x6kjavjfh5cnjegtlj;/home/axom/axom_tpls/clang-10.0.0/gmake-4.4.1-ovbffagtyrtthibtg7dm4xghwjwejn2f;/home/axom/axom_tpls/clang-10.0.0/blt-0.5.3-6jop7ew6gg3noxponfcbnxswwckmjpyc" CACHE PATH "")
```

Comment:
What's causing the new `CMAKE_PREFIX_PATH` requirement?

Is it the change to the spack `CachedCMakePackage` class?

Reply:
If they're not strictly necessary, should we add a function to selectively remove cache entries we don't want/need?

Reply:
This was a change in `CachedCMakePackage`. The idea was to get closer to mimicing how Spack builds packages outside of Spack.